### ISSUE TITLE
v5 not omitting null/undefined tag attribute 

### ISSUE THREAD

**Issue by themgoncalves** [jantimon/html-webpack-plugin#1597]
## Current behaviour 💣

The issue is related to the output of html tag's attribute with null value, which was being suppressed in the previous versions. 

This behaviour is particularly not wanted, and at some extension, harmful as in my case.

My application is set to use and validate `Sub Resource Integrity` (aka `integrity` attribute), which by definition should always contain a value, otherwise, it must be suppressed from the output.

```html
<script defer="defer" src="https://www.googletagmanager.com/gtag/js?id=[GA_ID]" integrity="null" crossorigin="anonymous"></script>
```
## Expected behaviour ☀️

Suppression of _null/undefined_ tags, as on v4.

```html
<script defer="defer" src="https://www.googletagmanager.com/gtag/js?id=[GA_ID]" crossorigin="anonymous"></script>
```

### Reproduction Example 👾

This is the base setup:

```js
    new SriPlugin({
      hashFuncNames: ['sha256', 'sha384', 'sha512'],
      enabled: isProd,
    }),
    new HtmlWebpackPlugin({
      template: './src/index.html',
      title: pkg.title,
      inject: true,
      minify: {
        collapseWhitespace: true,
        preserveLineBreaks: false,
      },
    }),
    new HtmlWebpackTagsPlugin({
      append: false,
      usePublicPath: false,
      scripts: [
        {
          path: `https://www.googletagmanager.com/gtag/js?id=${GOOGLE_ANALYTICS_ID || 'foo'}`,
          attributes: { async: true, integrity: false },
        },
      ],
    }),
```

## Environment 🖥

- `Node.js`: v14.11.0
- `darwin`: 20.3.0
- `webpack`: v5.21.2
- `webpack-subresource-integrity`: v1.5.2
- `html-webpack-tags-plugin`: v3.0.0

And of course, `html-webpack-plugin`: v5.

**Comment by jantimon** [comment]
the reason is this line: https://github.com/jantimon/html-webpack-plugin/blob/33d69f49e6e9787796402715d1b9cd59f80b628f/lib/html-tags.js#L33

**Comment by themgoncalves** [comment]
Terrific! Thank you for the quick reply @jantimon.

I will work on the change and create the PR for it 💪

**Comment by jantimon** [comment]
released as html-webpack-plugin@5.1.0 🎉

thank you so much for your help!
